### DISCUSSION TITLE
Camera Movement. NEED HELP!!

### DISCUSSION THREAD

**Discussion by thecampingcodesman** [webcamoid/webcamoid#557] [Q&A]
Sorry if this was already covered, but I wasnt able to find anything on it.
So I am running Webcamoid on Debian and am about to setup a full security network. The problem is most of my webcams are dome style and might require movement of each one to get a better view. Is there an easy way to in the program options to use the movement feature, or am I going to need to write my own code for that? Thanks in advance, any help would be greatly appreciated.

**Comment by hipersayanX** [comment]
I don't have a dome camera so I can't test it, if Webcamoid does not show any control for that, you can try implementing it.
